generator/generate-one.sh

```diff
@@ -43,37 +45,62 @@ fi
 
 cd googleapis
 
+## If $googleapis_folder does not exist,
+# try "google/$client_lib_name/v1"
+#     "google/devtools/$client_lib_name/v1"
+#     "google/cloud/($client_lib_name without 'cloud' prefix)/v1"
+# exit if none exists
+if [ ! -d "$googleapis_folder" ]
```

Comment:
We may need some logic to handle libraries that have other versions than 1. For example: `//google/devtools/clouddebugger/v2`

Reply:
I am moving the "guess the folder name" part to #1322